### DISCUSSION TITLE
Welcome to entra-powershell Discussions!

### DISCUSSION THREAD

**Discussion by SteveMutungi254** [microsoftgraph/entra-powershell#204] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Andre-DuToit** [comment]
Hi Steve,

Do we know when this powershell module will be officially released?

**Reply by SteveMutungi254** [reply]
@Andre-DuToit : We are now GA - from end of January, 2025!

Docs - https://aka.ms/entra/ps/docs
PowerShell gallery - https://aka.ms/entra/ps/gallery
